src/Services/LightningService.cs

```diff
@@ -674,9 +682,7 @@ await foreach (var response in openStatusUpdateStream.ResponseStream.ReadAllAsyn
             {
```

Comment:
Should this check that there are M of N signed instead of checking if there's one more than before?